parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -413,7 +413,8 @@ def worker_watchdog(self, kill_event: threading.Event):
                             raise WorkerLost(worker_id, platform.node())
                         except Exception:
                             logger.info("Putting exception for executor task {} in the pending result queue".format(task['task_id']))
-                            result_package = {'type': 'result', 'task_id': task['task_id'], 'exception': serialize(RemoteExceptionWrapper(*sys.exc_info()))}
+                            result_package = {'type': 'result', 'task_id': task['task_id'], 'exception':
+                                              serialize(RemoteExceptionWrapper(*sys.exc_info()))}
```

Comment:
Hi. Thanks for your PR.

For style, I would maybe write this with on three different lines - so that each key and value is next to each other on its own line, like this:

type: result
task_id: ....
exeception:  serialize(...)

That might make this easier to read when someone looks at this code later.

Reply:
Hello @benclifford thanks for the suggestion. I should have figured that out.